unitTests/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValueTest.ts

```diff
@@ -102,6 +102,34 @@ export function DynamicFacetValueTest() {
       expect(dynamicFacetValue.valueCaption).toBe(captionValue);
     });
 
+    it(`when a display value isn't specified
+      when there is no valueCaption
+      should return the value as display value`, () => {
+      expect(dynamicFacetValue.displayValue).toBe(dynamicFacetValue.value);
+    });
+
+    it(`when a display value is specified
+      when using the valueCaption with an object that contains the original value
+      should still return the original display value`, () => {
+      const displayValue = 'display value';
+      options = { valueCaption: { [dynamicFacetValue.value]: 'value caption' } };
+      initializeComponent();
+      const mockValue = DynamicFacetTestUtils.createFakeFacetValues(1)[0];
+      dynamicFacetValue = new DynamicFacetValue({ ...mockValue, displayValue }, facet);
+
+      expect(dynamicFacetValue.displayValue).toBe(displayValue);
+    });
+
+    it(`when a display value is not specified
```

Comment:
Related to the above comments, why would a display value not be specified?

Reply:
Only specified for the Search & Ranges for now